integration-tests/redis/src/main/java/io/quarkus/redis/it/ReactiveCommand.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.redis.it;
+
+import io.lettuce.core.dynamic.Commands;
+import io.lettuce.core.dynamic.annotation.Command;
+import io.reactivex.Single;
+import reactor.core.publisher.Mono;
```

Comment:
I think this is going to be an issue.

I think we want a unified reactive strategy and using reactor is not part of it.

@cescoffier @FroMage what's your take on this? Should we drop the reactive support of this PR for now? And work on supporting it later?